rpcserver.go

```diff
@@ -2535,7 +2535,27 @@ func (r *rpcServer) FetchAssetMeta(ctx context.Context,
 	}, nil
 }
 
-func marshalUniID(id universe.Identifier) *unirpc.ID {
+// MarshalUniProofType marshals the universe proof type into the RPC
+// counterpart.
+func MarshalUniProofType(
+	proofType universe.ProofType) (unirpc.ProofType, error) {
+
+	switch proofType {
+	case universe.ProofTypeUnspecified:
+		return unirpc.ProofType_PROOF_TYPE_UNSPECIFIED, nil
+	case universe.ProofTypeIssuance:
+		return unirpc.ProofType_PROOF_TYPE_ISSUANCE, nil
+	case universe.ProofTypeTransfer:
+		return unirpc.ProofType_PROOF_TYPE_TRANSFER, nil
+
+	default:
+		return unirpc.ProofType_PROOF_TYPE_UNSPECIFIED,
```

Comment:
nit: same here re returning an explicit `0` in error cases, would shorten the call as well.